storage/io/src/main/java/org/openscience/cdk/io/SDFWriter.java

```diff
@@ -67,29 +68,38 @@
     private Set<String>               propertiesToWrite;
 
     /**
-     * Constructs a new SDFWriter that writes to the given {@link Writer}.
+     * Create an SDfile writer that will output directly to the provided buffered writer.
      *
-     * @param   out  The {@link Writer} to write to
+     * @param wtr writer
      */
-    public SDFWriter(Writer out) {
-        if (out instanceof BufferedWriter) {
-            writer = (BufferedWriter) out;
-        } else {
-            writer = new BufferedWriter(out);
-        }
+    public SDFWriter(BufferedWriter wtr) {
+        this.writer = wtr;
         initIOSettings();
     }
 
     /**
-     * Constructs a new MDLWriter that can write to a given
-     * {@link OutputStream}.
+     * Create an SDfile writer, the provided writer is buffered
+     * if it's not an instance of BufferedWriter. For flush control
+     * etc please create with {@link BufferedWriter}.
+     *
+     * @param wtr writer
+     */
+    public SDFWriter(Writer wtr) {
+        this(ensureBuffered(wtr));
+        initIOSettings();
+    }
+
+    /**
+     * Create an SDfile writer, the provided output stream is wrapped
+     * in a UTF-8 buffered writer.
      *
-     * @param   output  The {@link OutputStream} to write to
+     * @param output out stream
      */
     public SDFWriter(OutputStream output) {
-        this(new OutputStreamWriter(output));
+        this(new OutputStreamWriter(output, StandardCharsets.UTF_8));
```

Comment:
Previously it was system dependant! Since all cheminf formats are ASCII and thus a subset of UTF-8 it only matters for aux fields like a title. e.g. 'Benzene Kékule Form' will now be correctly written on all systems.